lib/committee/drivers/open_api_2.rb

```diff
@@ -17,6 +17,14 @@ def default_query_params
       true
     end
 
+    def default_coerce_date_times
+      false
+    end
```

Comment:
Would you mind trying to see if we can keep all these methods ordered alphabetically in each file? I think that will just involve moving `default_coerce_date_times` to the top position above `default_coerce_form_params`.